docs/api/app.md

```diff
@@ -291,25 +315,25 @@ Sets the string to be displayed in the dock’s badging area.
 
 **Note:** This API is only available on OS X.
```

Comment:
Yeah that sounds good. I'll add a note to the top that explains platform specific APIs :+1: